packages/manager/core/generator/app/templates/src/routes/routes.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { RouteObject } from 'react-router-dom';
+import NotFound from '../pages/404';
+import { urls } from './routes.constant';
+
+const lazyRouteConfig = (importFn: CallableFunction): Partial<RouteObject> => {
+  return {
+    lazy: async () => {
+      const { default: moduleDefault, ...moduleExports } = await importFn();
+      return {
+        Component: moduleDefault,
+        ...moduleExports,
+      };
+    },
+  };
+};
+
+export const Routes: any = [
+  {
+    path: urls.root,
+    ...lazyRouteConfig(() => import('@/pages/layout')),
+    children: [
+      {
+        id: 'listing',
+        path: urls.listing,
+        ...lazyRouteConfig(() => import('@/pages/listing')),
+      },
+      {
+        path: urls.dashboard,
+        ...lazyRouteConfig(() => import('@/pages/dashboard/_layout')),
```

Comment:
can we rename this file index.tsx instead of _layout ?

Reply:
Sure It can be change 👍